app/controllers/api/concepts_controller.rb

```diff
@@ -0,0 +1,18 @@
+module API
+  class ConceptsController < BaseController
+    skip_before_action :authenticate_user!
+    before_action :authenticate_user
+
+    def show
+      track = Track.find_by!(slug: params[:track_id])
+      concept = track.concepts.find_by!(slug: params[:id])
+
+      render json: {
```

Comment:
If you're just going to prerender the blurb, I recommend using content negotiation here to just return the HTML. removes a _lot_ of logic required.